_posts/2023-10-05-cockpit-302.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 302
+author: lis
+date: '2023-10-05'
+tags: cockpit
+slug: cockpit-302
+category: release
+summary: 'Partition resizing'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 302 and cockpit-podman 78:
+
+## Storage: Partitions can be resized
+
+Cockpit can now also resize partitions, including their content such as filesystems or LUKS containers.
+
+![Storage page with shrink and grow buttons](/images/302-partition-resize.png)
+
+## Podman: Show label for Toolbox and Distrobox containers
+
+Podman now shows a visual label in the list of containers to differentiate Toolbox and Distrobox containers from other containers.
```

Comment:
Please add links to https://containertoolbx.org/ and https://github.com/89luca89/distrobox ?  Not sure if the latter is the most "official" homepage, but it's **not** https://www.distrobox.co/ :grin: